### PR TITLE
 project-mongo-api by Irina Jekabsone

### PR DESCRIPTION
**Pull request by Irina-web-dev**
You can find this API live on Heroku: https://books-base.herokuapp.com/

### CODE REVIEW

**Review by NoemiSalome — COMMENTED** [review]
Hej Irina!
I was supposed to do Tina's Code, but since we haven't heard from her I guess I'll be doing yours this week - so it was not my fault! Please talk to the manager about this and not me 💁🏼‍♀️ 😉
There wasn't too much code to review this week. But what I saw looks clean, pretty and totally readable! Great job on this! 🤩
On Heruko however I'm getting "Cannot GET /", so you are hiding your pretty endpoints behind that error. Which makes the Sherlock in me suspicious and wondering what's going on behind the scenes 🕵🏼‍♀️ Some puppy or money trading maybe? (I'd be interested in a puppy by the way 🐶)
I think you might be missing the endpoint "/", but I left a comment below. Hope that will solve it.

However I hope you enjoy the backend! It is giving me a bit of a hard time. I am hoping for better times 😅

**Review by NoemiSalome — CHANGES_REQUESTED** [review]
Hej Irina!
I was supposed to do Tina's Code, but since we haven't heard from her I guess I'll be doing yours this week - so it was not my fault! Please talk to the manager about this and not me 💁🏼‍♀️ 😉
There wasn't too much code to review this week. But what I saw looks clean, pretty and totally readable! Great job on this! 🤩
On Heruko however I'm getting "Cannot GET /", so you are hiding your pretty endpoints behind that error. Which makes the Sherlock in me suspicious and wondering what's going on behind the scenes 🕵🏼‍♀️ Some puppy or money trading maybe? (I'd be interested in a puppy by the way 🐶)
You might be missing the endpoint "/", but I left a comment below. 
However I hope you are enjoying the backend! 🤩

File: server.js [hunk 1]
```diff
@@ -1,35 +1,98 @@
...
+}
+
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
+
```
[the remark this answers is not in the document]
Comment (Irina-web-dev) [reply server.js:52]: Thank you @NoemiSalome 😊 I totally forgot about express list endpoints 🤷‍♀️ Now it should work properly ✔

File: server.js [hunk 2]
```diff
@@ -1,35 +1,103 @@
...
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+//Creating schema with data types
+const bookSchema = new mongoose.Schema({
+  bookID: Number,
```
Comment (KarolCislowski) [inline server.js:15]: I think it is no need to send this field to the database when you are using autogenerated _id 🤓

File: server.js [hunk 3]
```diff
@@ -1,35 +1,103 @@
...
+    res.json(booksToSend)
+  }
+
+  if (language_code) {
+    const booksToSend = await Book.find({
+      language_code: {
+        $regex: new RegExp(language_code, 'i')}
+    })
+    res.json(booksToSend)
+  }
```
Comment (KarolCislowski) [inline server.js:83]: sweet 🥳

**Review by KarolCislowski — APPROVED** [review]
Nice and clean API 🥳

**Pull request closed without merging**